provider/aws/go.mod

```diff
@@ -306,3 +251,7 @@ require (
 	mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b // indirect
 	mvdan.cc/unparam v0.0.0-20220706161116-678bad134442 // indirect
 )
+
+replace github.com/nitrictech/nitric/core => ../../core
+
+replace github.com/nitrictech/nitric/provider/common => ../common
```

Comment:
we probably should break common into small focused packages

Reply:
yes, but probably later (at the moment it only contains a single source file :) )